DonjinKrawler-Client/src/main/java/donjinkrawler/items/Armor.java

```diff
@@ -41,6 +42,11 @@ public Armor clone() throws CloneNotSupportedException {
         return armor;
     }
 
+    @Override
+    public void accept(ItemVisitor v) {
```

Comment:
gal gėriau v -> visitor?

From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Use Intention-Revealing names.`